modules/support_ticket/src/Tests/SupportTicketCreationTest.php

```diff
@@ -49,7 +49,7 @@ function testTicketCreation() {
     $edit = array();
     $edit['title[0][value]'] = $this->randomMachineName(8);
     $edit['body[0][value]'] = $this->randomMachineName(16);
-    $edit['field_priority'] = 'high';
+    $edit['field_priority'] = '120'; // 120 = 'high'
```

Comment:
That was almost certainly a mistake on my part. IMO it should be an integer everywhere.